src/data/fakeData.js

```diff
@@ -204,6 +204,48 @@ export const fakeActivityData = [{
     "minutesActive": 138,
     "flightsOfStairs": 10
   },
+  {
+    "userID": 3,
+    "date": "2019/06/09",
+    "numSteps": 7402,
+    "minutesActive": 116,
+    "flightsOfStairs": 33
+  },
+  {
+    "userID": 3,
+    "date": "2019/06/11",
+    "numSteps": 7402,
+    "minutesActive": 116,
+    "flightsOfStairs": 33
+  },
+  {
+    "userID": 3,
+    "date": "2019/06/10",
+    "numSteps": 7402,
+    "minutesActive": 116,
+    "flightsOfStairs": 33
+  },
+  {
+    "userID": 3,
+    "date": "2019/06/13",
+    "numSteps": 7402,
+    "minutesActive": 116,
+    "flightsOfStairs": 33
+  },
+  {
+    "userID": 3,
+    "date": "2019/06/12",
+    "numSteps": 7402,
+    "minutesActive": 116,
+    "flightsOfStairs": 200
+  },
+  {
+    "userID": 3,
+    "date": "2019/06/14",
+    "numSteps": 7402,
+    "minutesActive": 116,
+    "flightsOfStairs": 33
+  },
```

Comment:
woowwweee

Reply:
I had to add a week's worth of activities for one user in order to test methods that use a week's activities 😅